packages/assistify-dbs-translation/server/methods/translate.js

```diff
@@ -0,0 +1,13 @@
+
+Meteor.methods({
+	'assistify.translate'(httpMethod, url, options) {
```

Comment:
This is a *very* generic signature which has got nothing to do with autotranslate. Also, it could be used to inject malware from another server (the client could make the server `get` from any address). 
There should be a limitation on the URL.